payara/Payara
# FISH-7721 FISH-7723 Make Certificate Management an Extension

## Description
Removes the Certificate Management code from the Community repo, with us having finally made it the extension it was always meant to be: https://github.com/payara/Certificate-Management/tree/Payara6

## Important Info
### Blockers
None

## Testing
### New tests
None

### Testing Performed
* Created a temporary maven repository to ensure nothing cached: `mkdir D:\Downloads\FISH-7721`
* Built [Certificate Management](https://github.com/payara/Certificate-Management/tree/Payara6) 2.0.0-SNAPSHOT (Payara6 branch):
  * `git clone https://github.com/payara/Certificate-Management`
  * `git checkout Payara6`
  * `mvn clean install -Dmaven.repo.local=D:\Downloads\FISH-7721`
* Built Payara Server: `mvn clean install -PBuildEmbedded -DskipTests -T 1C -Dmaven.repo.local=D:\Downloads\FISH-7721`
* Run CLI command without server running to ensure it still works: `asadmin list-truststore-entries`
* Start server (everything should be bon): `asadmin start-domain --verbose --debug`
* Run an admin command to ensure it still works: `asadmin list-certificates`
* Load admin console and ensure console integration present: _localhost:4848 -> server -> Certificate Management_

### Testing Environment
Windows 11, Zulu JDK 11.0.20

## Documentation
https://github.com/payara/Payara-Documentation/pull/309

## Notes for Reviewers
I will deploy a SNAPSHOT to Nexus to allow Jenkins to test, you may also wish to make use of it if you wish to skip the building of Certificate Management for testing.

There Shouldn't™ be any issues with OSGi - as an extension it only _takes_ from the server (with me setting the internals version to the previously published 6.5.0 so the import range is [6.5,7) and the packages it exports are only used by itself.
